opensrp-reporting/src/main/java/org/smartregister/reporting/repository/DailyIndicatorCountRepository.java

```diff
@@ -108,6 +108,30 @@ public void add(@Nullable CompositeIndicatorTally indicatorTally) {
         database.insert(Constants.DailyIndicatorCountRepository.INDICATOR_DAILY_TALLY_TABLE, null, createContentValues(indicatorTally));
     }
 
+    public void updateIndicatorValue(String id, String Value)
+    {
+        SQLiteDatabase database = getWritableDatabase();
+        Object[] queryArgs = {
+                Constants.DailyIndicatorCountRepository.INDICATOR_DAILY_TALLY_TABLE,
+                Constants.DailyIndicatorCountRepository.INDICATOR_VALUE,
+                Constants.DailyIndicatorCountRepository.ID,
+        };
+
+        Cursor cursor = null;
+        try{
+            cursor = database.rawQuery(String.format("UPDATE %s SET %s = "+Value+" WHERE %s = "+ id, queryArgs),null);
```

Comment:
Use query placeholders to prevent SQL injection

Reply:
correction done